### PR TITLE
Csr hierarchy

### PR DESCRIPTION
**Pull request by ismailsunni**
I added `with-parent` parameter to retrieve all of the parent's layer value recursively. It only works for single layer key to avoid complexity... It is also disabled by default.

Sample output:
```
[
    {
        "key": "quaternary_catchment_area",
        "parent": "tertiary_catchment_area",
        "value": "D13J",
        "display_name": "Quaternary Catchment Area",
        "description": null,
        "query_type": "WFS"
    },
    {
        "key": "tertiary_catchment_area",
        "parent": "secondary_catchment_area",
        "value": "D13",
        "display_name": "Tertiary Catchment Area",
        "description": null,
        "query_type": "WFS"
    },
    {
        "key": "secondary_catchment_area",
        "parent": "primary_catchment_area",
        "value": "D1",
        "display_name": "Secondary Catchment Area",
        "description": null,
        "query_type": "WFS"
    },
    {
        "key": "primary_catchment_area",
        "parent": null,
        "value": "Region D",
        "display_name": "Primary Catchment Area",
        "description": null,
        "query_type": "WFS"
    }
]
```

### CODE REVIEW

File: django_project/base/management/commands/load_service_registry_data.py [hunk 1]
```diff
@@ -74,6 +76,27 @@ def handle(self, *args, **options):
                     failed_load += 1
             else:
                 incomplete_item += 1
+        print('Set parent')
```
Comment (timlinux) [inline django_project/base/management/commands/load_service_registry_data.py]: naughty :-P

**Comment by timlinux** [comment]
As discussed with @ismailsunni we need to model the hierarchy  like this:

screen shot 2018-06-26 at 11 47 47

**Comment by ismailsunni** [comment]
Hi @timlinux 
An update, I have made new models based on the diagram, and simple admin to do ordering or another simple use case (CRUD). Sample UI (thanks to Grapelli, life is easier):

geocontext_grapelli

I will continue the API to retrieve per group/collection in the other PR to make the PR smaller.

**Review by dimasciput — APPROVED** [review]
LGTM

**Comment by ismailsunni** [comment]
Covered in #9

**Pull request closed without merging**